pyanaconda/ui/gui/spokes/welcome.py

```diff
@@ -89,6 +90,9 @@ def apply(self):
         if flags.flags.automatedInstall and not geoloc.geoloc.enabled:
             return
 
+        if not is_module_available(TIMEZONE):
+            return
```

Comment:
This would be better to refactor in the future. I don't think it's wise to have code .. conditions with early return ... code again. It's pretty simple to add code at the bottom but miss the early returns in the middle.

Also shouldn't we merge these two conditions. They are both disabling geoloc at the end.

Reply:
I will change it in upstream. This code should be in a separate function.